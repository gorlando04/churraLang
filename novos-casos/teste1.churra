amigo "A" contribui 100
amigo "B" contribui 100
item comida preco_unit 50 quantidade 4
rateio igual