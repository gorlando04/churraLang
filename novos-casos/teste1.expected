Total do churrasco: R$200.00
A deve pagar: R$100.00
B deve pagar: R$100.00